keywords/TestServeriOS.py

```diff
@@ -73,12 +91,18 @@ def download(self, version_build=None):
         # Package not downloaded, proceed to download from latest builds
         downloaded_package_zip_name = "{}/{}".format(BINARY_DIR, self.package_name)
         if self.platform == "ios":
-            url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+            if self.build is None:
+                if self.version < "2.0.2":
```

Comment:
both if and else stmts are same, why is this if condition required?

Reply:
not they are not - "couchbase-lite/ios" v/s "couchbase-lite-ios"